guess-the-country/country-backend/country/api.py

```diff
@@ -108,7 +109,7 @@ async def explain_api(file: UploadFile = File(...)):
     image = load_image(file)
     pre_image = preprocess(image)
 
-    explanation = explain_cnn(pre_image, model)
+    explanation = explain_cnn(pre_image)
     explain_id = uuid.uuid4()
     encoded_image_string = convert_explanation(explanation)
     encoded_bytes = bytes("data:image/png;base64,",
```

Comment:
Auch hier: Diese Logik sollte in ein eigenes Modul, hier in `api.py` sollte nur durchgereicht werden. Dann kann man die Erklärungen individuell testen.